gateway/1.6/modules/ROOT/pages/flex-troubleshoot-regex.adoc

```diff
@@ -0,0 +1,10 @@
+= Troubleshoot Compatibility Issues with Regular Expressions
+
+Troubleshoot compatibility issues with using regular expressions:
```

Comment:
I stumbled a bit over the wording "with using." Maybe revise to something like this: 

Troubleshoot compatibility issues that occur when using regular expressions

Reply:
Good point! Fixed!